pkg/controller/clusterstate/api/resource_info.go

```diff
@@ -84,6 +84,20 @@ func NewResource(rl v1.ResourceList) *Resource {
 	return r
 }
 
+func GetCondition(nodeStatus v1.Node) v1.ConditionStatus {
+	var currentNodeStatus v1.ConditionStatus = v1.ConditionFalse
```

Comment:
Suggest default to `ConditionTrue` similar to what is defined in the default above in the creation.

Reply:
Done, thanks